### PR TITLE
Make TextInput ignore composition event on mWeb

### PR DESCRIPTION
**Pull request by SzymczakJ**
### Related Issues

https://github.com/Expensify/App/issues/42523

### CODE REVIEW

**Comment by SzymczakJ** [comment]
I have read the CLA Document and I hereby sign the CLA

File: src/web/browserUtils.ts
Comment (tomekzaw) [inline src/web/browserUtils.ts:3]: How did you come up with this line? Is this copied from somewhere (from existing codebase) or newly added code snippet?

Comment (SzymczakJ) [reply src/web/browserUtils.ts:3]: I copied it from Expensify App. https://github.com/Expensify/App/blob/90dee7accae79c49debf30354c160cab6c52c423/src/libs/Browser/index.website.ts#L41

Comment (tomekzaw) [reply src/web/browserUtils.ts:3]: Good, can we please add a comment in the code with this URL where it was copied from?

Comment (tomekzaw) [reply src/web/browserUtils.ts:9]: Thanks

**Review by tomekzaw — APPROVED** [review]
LGTM but let's wait for @Skalakid and @BartoszGrajdek's approval.

**Comment by Skalakid** [comment]
LGTM 👍

**Review by Skalakid — APPROVED** [review]
LGTM 👍

**Review by BartoszGrajdek — APPROVED** [review]
LGTM!

**Pull request merged**
